source/main_cr.f90

```diff
@@ -1290,13 +1277,8 @@ end subroutine envarsv
           pstop (j) = .false.
 
 !         values related to momentum
-!         old proton only terms:
-!          ejv   (j) = sqrt(ejfv(j)**2+pma**2)
-!          dpsv  (j) = (ejfv(j)-e0f)/e0f
-!          oidpsv(j) = one/(one+dpsv(j))
-
-          ejv   (j)   = sqrt(ejfv(j)**2+nucm(j)**2)              ! hiSix
-          dpsv  (j)   = (ejfv(j)*(nucm0/nucm(j))-e0f)/e0f         ! hiSix
+          ejv   (j)   = sqrt(ejfv(j)**2+nucm(j)**2)
+          dpsv  (j)   = (ejfv(j)*(nucm0/nucm(j))-e0f)/e0f
           oidpsv(j)   = one/(one+dpsv(j))
           mtc     (j) = (nzz(j)*nucm0)/(zz0*nucm(j))
           moidpsv (j) = mtc(j)*oidpsv(j)
```

Comment:
I guess that here we should also initialise `dpd(j)` and `dpsq(j)`, as done in the energy update function, though I don't know exactly what these variables are for... and set suitable values for the on-momentum particles...